MIDI/mathphreak_Learningbaby.jsfx

```diff
@@ -1,15 +1,19 @@
-version: 1.0
+version: 1.0.1
```

Comment:
Bumping the version number (and changelog) is not required to change the value of the about tag because it's tied to the entire package rather than to a specific version (see Package Tags vs Version Tags in the [packaging documentation](https://github.com/cfillion/reapack-index/wiki/Packaging-Documentation#metadata)).